### PR TITLE
Don't show History of changes if there is no any

### LINKED ISSUE #419: Don't show History of changes if there is no any

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Bug Fix

## Related issues

Resolve #419

### CODE REVIEW

**Comment by Katrin-kudryash** [comment]
Untitled 11

**Comment by Katrin-kudryash** [comment]
Untitled 15

**Review by nyet-ty — APPROVED** [review]

File: src/components/ProblemHistoryCard/ProblemHistoryCard.tsx [hunk 1]
```diff
@@ -38,6 +39,7 @@ const VisibleContainer = styled.div`
 
 const Content = styled.div`
     margin-top: ${gapSm};
+    max-width: 650px;
```
Comment (9teen90nine) [inline src/components/ProblemHistoryCard/ProblemHistoryCard.tsx:42]: let's make it a bit wider, like 1000px or so

**Comment by Katrin-kudryash** [comment]
Untitled 4

File: src/components/Problem/Problem.tsx [hunk 2]
```diff
@@ -112,21 +112,24 @@ export const Problem: FC<ProblemProps> = ({ problem }) => {
...
-                {tr('History of changes')}
-
-                {isProblemHistoryExpanded ? (
-                    <IconArrowUpSmallOutline size="m" />
-                ) : (
-                    <IconArrowDownSmallOutline size="m" />
-                )}
-            </StyledTitle>
-            {problem.problemHistory.length > 0 && isProblemHistoryExpanded && (
+            {problem.problemHistory.length > 0 && (
```
Comment (tsumo) [inline src/components/Problem/Problem.tsx:115]: Nullable can also check for zero

**Review by tsumo — CHANGES_REQUESTED** [review]

**Review by tsumo — APPROVED** [review]

**Review by 9teen90nine — APPROVED** [review]

**Pull request merged**